pulsar-common/src/main/java/org/apache/pulsar/common/conf/InternalConfigurationData.java

```diff
@@ -23,6 +23,7 @@
 
 public class InternalConfigurationData {
 
+    public final static int MESSAGE_META_SIZE = 10 * 1024;
```

Comment:
This class is a POJO used to serialize to JSON. 

I'd keep this constant in `Commands` where all the binary protocol stuff is. 
eg: `Commands.MESSAGE_SIZE_FRAME_PADDING` 

along with `Commands.DEFAULT_MAX_MESSAGE_SIZE`

Reply:
Ok, will fix